receiver/apachedruidreceiver/metadata.yaml

```diff
@@ -0,0 +1,2541 @@
+type: apachedruid
+
+status:
+  class: receiver
+  stability:
+    development: [metrics, logs]
+  distributions: []
+  codeowners:
+    active: [atoulme, yuanlihan]
+
+resource_attributes:
```

Comment:
this is a really great achievement to have condensed and described all those metrics. Can we try to land this PR without them first and have a second PR to discuss them? It will make things easier. We can add in segments as well as you implement support.

Reply:
@atoulme Thanks for your suggestion. Yes, it makes more sense to discuss those metrics in the second PR. Updated.